presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -305,6 +306,11 @@ public SystemSessionProperties(
                         featuresConfig.isExchangeCompressionEnabled(),
                         false),
                 booleanSessionProperty(
+                        EXCHANGE_BELOW_OUTPUT,
+                        "Exchange node added below Output node in the plan",
```

Comment:
IMO this is a very low level description. I would say something about offloading calculation out of coordinator.